addons/calendar/views/calendar_views.xml

```diff
@@ -426,10 +433,10 @@
                        options="{'icon': 'fa fa-link', 'shouldOpenRecord': true}"/>
                 <field name="alarm_ids" invisible="not alarm_ids" options="{'icon': 'fa fa-bell-o'}"/>
                 <field name="categ_ids" invisible="not categ_ids" options="{'icon': 'fa fa-tag', 'color_field': 'color'}"/>
-                <!-- For recurrence update Dialog -->
-                <field name="recurrency" invisible="1"/>
-                <field name="recurrence_update" invisible="1"/>
+                <field name="res_model_name" invisible="not res_model_name"
+                       options="{'icon': 'fa fa-link', 'shouldOpenRecord': true}"/>
                 <field name="partner_id" string="Organizer" options="{'icon': 'fa fa-user-o'}"/>
```

Comment:
`res_model_name` is a duplicate of the field a few lines above :nerd_face: please remove it ^^

Reply:
Also in the end we do not remove unused fields I believe? we can remove that part from the commit message